app/assets/stylesheets/pageflow/editor/files_explorer.css.scss

```diff
@@ -64,3 +63,61 @@
     @include icon-button();
   }
 }
+
+.spinner > div {
+  background-color: #ddd;
+  height: 100%;
+  width: 6px;
+  display: inline-block;
+
+  -webkit-animation: loading-animation 1.2s infinite ease-in-out;
+  animation: loading-animation 1.2s infinite ease-in-out;
+}
+
+.loading .spinner {
+  margin: 20px auto;
+  width: 50px;
+  height: 40px;
+  text-align: center;
+  font-size: 10px;
+
+  .rect2 {
+    -webkit-animation-delay: -1.1s;
+    animation-delay: -1.1s;
+  }
+
+  .rect3 {
+    -webkit-animation-delay: -1.0s;
+    animation-delay: -1.0s;
+  }
+
+  .rect4 {
+    -webkit-animation-delay: -0.9s;
+    animation-delay: -0.9s;
+  }
+
+  .rect5 {
+    -webkit-animation-delay: -0.8s;
+    animation-delay: -0.8s;
+  }
+}
+
+@-webkit-keyframes loading-animation {
+  0%, 40%, 100% {
+    -webkit-transform: scaleY(0.4)
+  }
+  20% {
+    -webkit-transform: scaleY(1.0)
+  }
+}
+
+@keyframes loading-animation {
+  0%, 40%, 100% {
+    transform: scaleY(0.4);
+    -webkit-transform: scaleY(0.4);
+  }
+  20% {
+    transform: scaleY(1.0);
+    -webkit-transform: scaleY(1.0);
+  }
+}
```

Comment:
There is a special folder for keyframe animations `app/assets/stylesheets/pageflow/animations` since they a defined in a shared global scope. Please use [bourbon mixins](http://bourbon.io/docs/#keyframes) to define keyframes.